src/iden3comm/handlers/message-handler.ts

```diff
@@ -131,10 +134,20 @@ export class MessageHandler {
 
     const response = await this.messageHandler.handle(message, context);
 
-    if (response) {
-      return this._params.packageManager.packMessage(unpackedMediaType, response, {});
+    if (!response) {
+      return null;
     }
 
-    return null;
+    let packerParams = {};
+    const senderDID = (context as { senderDID?: DID })?.senderDID;
+    if (unpackedMediaType === MediaType.ZKPMessage && senderDID) {
```

Comment:
Shouldn't it throw if `unpackedMediaType` is `MediaType.ZKPMessage` but no `senderDID` in context ?